test/connector/tcp/mssql/mssql_query.go

```diff
@@ -8,6 +8,8 @@ import (
 	"os/exec"
 	"text/tabwriter"
 
+	_ "github.com/denisenkom/go-mssqldb" // registers the "mssql" driver to database/sql
+
```

Comment:
How was this working previously?  (ie, seems like we would have always needed this...)